### ISSUE TITLE
Freezing, cramping, unresponsive issue when opening tab URL in sidebar

### ISSUE THREAD

**Issue by universish** [flathub/one.ablaze.floorp#108]
I added some website URLs to the sidebar. for each URL I created a separate container and set it to work in that container. when I click on the website icon in the sidebar, it works in that container. When it starts to run, it freezes excessively, it stutters excessively, it consumes CPU all the time, finally it becomes unresponsive. Then the unresponsive, wait for it to respond or close option window pops up. Even if you click wait for it to respond, this window pops up again. It keeps repeating. The browser becomes inoperable. 
OS: fedora 40

**Comment by dominichayesferen** [comment]
This sounds like a potentially upstream bug in Floorp itself.

Just in case it is, https://github.com/Floorp-Projects/Floorp/issues/new/choose

**Comment by universish** [comment]
https://github.com/Floorp-Projects/Floorp/issues/1494

**Comment by universish** [comment]
I was having this problem with the floorp flathub version. Yesterday I deleted flathub and started using portable. The vertical tab setting is on. In the vertical tab, in the settings; “Color the Tab Bar with the colors of the tab icon of the open site” is on. It seems like the portable version doesn't have this problem. 
Since I deleted the floorp flathub package, unfortunately I can't show or video this problem.
